src/app/components/chip/chip.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="tw-inline-flex tw-relative tw-w-full">
+  <ion-chip
+    class="chip tw-w-full tw-justify-center"
+    [class.chip--missing]="!isExists()"
+    [class.chip--collected]="isCollected()"
+    [class.chip--loading]="isLoading()"
+    [disabled]="!isExists() || isLoading()"
+    (click)="handleChipClick(chipId())">
+    <ion-label>{{ chipId() }}</ion-label>
+  </ion-chip>
+
+  <ng-container *ngIf="isLoading()">
+    <div class="tw-absolute tw-flex tw-justify-center tw-items-center tw-h-full tw-w-full">
```

Comment:
if there is chance - will be nice to avoid using absolute position or if it is not possible - make sure items positioned according to all edge points (top, bottom, left, right)